unicodetools/org/unicode/propstest/CheckEmojiProps2.java

```diff
@@ -0,0 +1,30 @@
+package org.unicode.propstest;
+
+import org.unicode.props.IndexUnicodeProperties;
+import org.unicode.props.UcdProperty;
+import org.unicode.props.UcdPropertyValues.Binary;
+
+import com.google.common.collect.Sets;
+import com.ibm.icu.text.UnicodeSet;
+
+/**
+ * PR #44 added this class, but it was obviously added under the same name with
```

Comment:
PR #44 is this one. Were you trying to refer to a different one? Otherwise I am confused about the "Re-adding" comment.

Reply:
1. in Feb 2021 Mark added org.unicode.propstest.CheckEmojiProps in 968c94c7f2c600027709062dabe52aa830004ed9 in this PR#44
2. in June 2021 Mark also added org.unicode.propstest.CheckEmojiProps in 163def1d820fae750b11a7f33cf01dd1959f6b35 in #65  (merged) with different content
3. There's also org.unicode.tools.CheckEmojiProps which I haven't evaluated.

So, I was meaning to leave the comment there as a signpost to that perhaps someone in the future could make sense of things.

Reply:
Much clearer. Please add all of these details to the comment.